packages/@aws-cdk/aws-appmesh/lib/virtual-gateway.ts

```diff
@@ -60,6 +61,13 @@ export interface VirtualGatewayBaseProps {
    * @default - no access logging
    */
   readonly accessLog?: AccessLog;
+
+  /**
+   * Default Configuration Virtual Node uses to communicate with Virtual Service
+   *
+   * @default - No Config
+   */
+  readonly backendDefaults?: ClientPolicy;
```

Comment:
Why this mismatch between the property name, and the type name? This reads a little surprising to me... Will `ClientPolicy` be used for something else other than `backendDefaults`?

Reply:
ClientPolicy is being used for [backends](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-appmesh-virtualnode-virtualservicebackend.html) and [backend defaults](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-appmesh-virtualnode-backenddefaults.html).

Reply:
That doesn't really answer my question 🙂.

It's very weird to have a property called `backendDefaults`, and its type being `ClientPolicy`.

I see `ClientPolicy` is also used in `VirtualNode`, where the property is called `backendDefaults` too, and in `VirtualService`, where it's called `clientPolicy`.

Can you explain why the weird naming?